tests/console/java.pm

```diff
@@ -37,20 +38,37 @@ sub run {
     # https://www.suse.com/releasenotes/x86_64/SUSE-SLES/15-SP2/#development-java-versions
     # java-11-openjdk                   -> Basesystem
     # java-10-openjdk & java-1_8_0-ibm  -> Legacy
-    my $cmd = 'install --auto-agree-with-licenses ';
-    $cmd .= (is_sle('15+') || is_leap) ? 'java-11-openjdk* java-1_*' : 'java-*';
-    zypper_call($cmd, timeout => 2000);
-
-    if (script_run 'rpm -q wget') {
-        zypper_call 'in wget';
+    if (check_var('TRANSACTIONAL_SERVER', '1')) {
+        select_console 'root-console';
+        assert_script_run("transactional-update --continue pkg install --no-confirm java-*", 2000);
+        check_reboot_changes;
+        # find snapshot number for rollback and reboot after update
+        my $snapshot = script_output "snapper list | tail -1 | cut -d'|' -f1 | tr -d ' *+'";
+        assert_script_run("transactional-update rollback $snapshot");
+        assert_script_run("transactional-update reboot");
```

Comment:
What's the `t-u reboot` + console resetting for?

Reply:
after transactional-update, reboot is required. this is designed for transactional server, otherwise the installed package cannot be used.

Reply:
`t-u reboot` doesn't actually reboot. It only sets a flag that it needs a reboot.

Reply:
checked this, for this test, rollback set correct snapshot which will be used for booting into it with java installation. transactional-update reboot does then reboot. You can check screen for this.

Reply:
The test reboots once, in `check_reboot_changes`.

After that, there is no reboot in this test anymore. That `t-u reboot` is a noop is visible here: http://10.162.23.47/tests/8516#step/java/36

Reply:
indeed, I tested it now for update. Thanks for you feedback!